rosidl_typesupport_protobuf_cpp/cmake/rosidl_typesupport_protobuf_cpp_generate_interfaces.cmake

```diff
@@ -181,12 +182,12 @@ add_dependencies(
 if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
   install(
     DIRECTORY "${_output_path}/"
-    DESTINATION "include/${PROJECT_NAME}"
+    DESTINATION "include/${PROJECT_NAME}/${PROJECT_NAME}"
```

Comment:
This duplication looks like a potential problem. I suspect that the downstream include should be using more fully qualified include strings aka "<PROJECT_NAME>/<HEADER_FILE_NAME.h>"